config.template

```diff
@@ -1,88 +1,109 @@
+# the following shortcuts exist:
+#   'all' -- all analyses will be run
+#   'all_timeseries' -- all time-series analyses will be run
+#   'all_modelvsobs' -- all analyses comparing model climatologies with
+#                       observatoins will be run
+#   'all_ocean' -- all ocean analyses will be run
+#   'all_seaice' -- all sea-ice analyses will be run
+#   'no_ohc_timeseries' -- skip the 'ohc_timeseries' (and similarly with the other analyses).
+#   'no_ocean', 'no_timeseries', etc. -- in analogy to 'all_*', skip the given category of analysis
+generate = ['all']
+
+# alternative examples that would perform all analysis except
+#   'ohc_timeseries'
+#generate = ['sst_timeseries', 'all_modelvsobs', 'all_seaice']
+#generate = ['all', 'no_ohc_timeseries']
+# Each subsequent list entry can be used to alter previous list entries. For
+# example, the following would produce all analyses except ocean model vs.
+# observations (albeit not in a very intuitive way):
+#generate = ['all', 'no_ocean', 'all_timeseries']
+
+[climatology]
+
+# the first year over which to average climotalogies
+start_year = 11
+# the last year over which to average climotalogies
+end_year = 20
+
+[time_series]
+
+# start and end years for timeseries analysis. Using out-of-bounds values
+#   like start_year = 1 and end_year = 9999 will be clipped to the valid range
+#   of years, and is a good way of insuring that all values are used.
+start_year = 1
+end_year = 9999
+
+[ocean_observations]
+
+# directory where ocean observations are stored
+basedir = /dir/to/ocean/observations
+sst_subdir = SST
+sss_subdir = SSS
+mld_subdir = MLD
+
+[ocean_reference]
+
+# directory where ocean reference simulation results are stored
+basedir = /dir/to/ocean/reference
+
+[seaice_observations]
+
+# directory where sea ice observations are stored
+basedir = /dir/to/seaice/observations
+areaNH = IceArea_timeseries/iceAreaNH_climo.nc
+areaSH = IceArea_timeseries/iceAreaSH_climo.nc
+volNH = PIOMAS/PIOMASvolume_monthly_climo.nc
+volSH = none
+
+[seaice_reference]
+
+# directory where sea ice reference simulation results are stored
+basedir = /dir/to/seaice/reference
 
 [time]
-# the first year over which to average climotologies
-climo_yr1 = 6
-# the last year over which to average climotologies
-climo_yr2 = 10
 # the offset year to be added to simulation years
 yr_offset = 1849
-# start and end years for timeseries analysis
-timeseries_yr1 = 1
-timeseries_yr2 = 9999
-
-[ohc_timeseries]
-generate = 1
-
-[sst_timeseries]
-generate = 1
-
-[nino34_timeseries]
-generate = 0
-
-[mht_timeseries]
-generate = 0
-
-[moc_timeseries]
-generate = 0
-
-[sst_modelvsobs]
-generate = 1
-
-[sss_modelvsobs]
-generate = 1
 
-[mld_modelvsobs]
-generate = 1
-
-[seaice_timeseries]
-generate = 1
+[remap]
 
-[seaice_modelvsobs]
-generate = 1
+# paths to mapping files (which will hopefully be eliminated soon)
+mpas_remapfile = /global/project/projectdirs/acme/mapping/maps/map_oEC60to30_TO_0.5x0.5degree_blin.160412.nc
```

Comment:
Yes, that's the reason for my comment.  I've been skipping seaice_modelvsobs in my testing because I don't have mapping files, so this is a serious problem.  I really hope my upcoming work on generalizing interpolation can get rid the need for this option.